src/view/dataElements/xdmObject/constants/alwaysDisabledFields.js

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+/**
+ * Fields that users should not be allowed to edit.
+ */
+export default [
+  "_id",
+  "identityMap",
+  "eventType",
```

Comment:
I brought this up with Justin as I was working on it and he wanted to force users to provide it through the Send Event action. I suspect he may not feel strongly about it though so I'll bring it up with him again.

Reply:
I don't feel strongly at all about this. There were two things that pushed me slightly this way. 1. It makes debugging/support easier since there is only one way to set it. 2. We can alway relax that restriction later. Again I don't feel strongly. Finally since we talked about this we have been batting around ideas for setting XDM via rules. If we go that route we would want to allow this to be set in different ways.